assets/css/admin.css

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Can we turn this annoying rule off?

Reply:
Not sure if I follow: the empty line was there on purpose?
We can get rid of the rule but it might affect non `*.css` files aswell. Need to look into it.